SlackBotPrototype/API/SqLitePersistence.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Linq;
+using SQLite;
+
+namespace API
+{
+	// SHORTCUT: Normally we'd break out the repositories into
+	// a file per entity. Keep them together for simplicity here.
+	// 
+	// Also skipping tests for these because we'd need to set up integration tests
+	// with a SQLite database. If we weren't using a embedded file DB like SQLite, testing
+	// against a DB could get complicated.
+	public class SqLitePersistence : IRepository
+	{
+
+		public User Get(string slackId)
+		{
+			if (string.IsNullOrWhiteSpace(slackId))
+			{
+				return null;
+			}
+
+			using (var db = new SQLiteConnection(ConfigConstants.DbName))
+			{
+				return db.Query<User>($"SELECT * FROM User WHERE SlackId = '{slackId}'").FirstOrDefault();
+			}
+		}
+
+		// SHORTCUT: We skip testing this because it's hard to test this in isolation, but it contains

```

Comment:
I was referencing the fact that I skipped integration tests using SQLite since this code directly interacts via SQL. Definitely doable and worthwhile considering logic in the query.